### PR TITLE
Re-activate and Update Lowes.com.xml

### PR DESCRIPTION
**Pull request by cschanaj**
#10782, #10873, #10900

### CODE REVIEW

File: src/chrome/content/rules/Lowes.com.xml [hunk 1]
```diff
@@ -1,122 +1,38 @@
...
+			 - dpmbapps.lowes.com
+			 - dpmbappsqa.lowes.com
+			 - mobileapps.lowes.com
+			 - sadclwscom.lowes.com
+
+		SSL peer certificate was not OK:
+			 - metrics.lowes.com
+
+		Status code mismatch (over Travis):
+			 - lowes.com
```
Comment (J0WI) [inline src/chrome/content/rules/Lowes.com.xml]: You white listed it, but you also redirect `^` to `www`.

**Comment by cschanaj** [comment]
@J0WI I removed the complicated rewrite for `^` and whitelisted the file again. thanks!

**Comment by J0WI** [comment]
Let's wait for an answer in #10944 with those strange status code mismatch issues.

**Comment by Hainish** [comment]
This is also an Akamai host, like #10944 

```shell
user@https-everywhere ~ $ dig +short m.lowes.com
23.197.96.66
user@https-everywhere ~ $ dig +short -x 23.197.96.66
a23-197-96-66.deploy.static.akamaitechnologies.com.
```

File: src/chrome/content/rules/Lowes.com.xml (hunk 1 above)
Comment (J0WI) [inline src/chrome/content/rules/Lowes.com.xml]: This comment is now obsolete.

**Pull request merged**